Trumpf.Coparoo.Web/Controls/Table/TableSegment.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Trumpf.Coparoo.Web.Controls.Table
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Generic table segment control object.
+    /// </summary>
+    abstract public class TableSegment : ControlObject
```

Comment:
According to [https://stackoverflow.com/questions/191929/is-there-a-convention-to-the-order-of-modifiers-in-c]() it's more common to write it as `public abstract`.